docs/user_guide/admission_controller.md

```diff
@@ -0,0 +1,208 @@
+| `schedulerServiceAddress` | “yunikorn-service:9080”                 | address of the YuniKorn scheduler service								   |
+
+`amServiceName` is required for the admission controller to register itself properly with Kubernetes, and should normally not be changed.
+
+`schedulerServiceAddress` must be reachable by the admission controller, and is used by the admission controller when validating ConfigMap changes. The admission controller will contact the REST API on the scheduler to validate any proposed ConfigMap changes. This setting should not normally be changed.
+
+:::note NOTE
+A change to `amServiceName` and `amServiceName` requires a restart of the YuniKorn admission controller to take effect.
+:::
+Filtering configuration, all entries start with the prefix `admissionController.filtering.`.
+
+
+| Variable             | Default value                         | Description                                                                							   |
+|----------------------|---------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `processNamespaces`  | ""                                    | which namespaces will have pods forwarded to YuniKorn for scheduling       							   |
+| `bypassNamespaces`   | “^kube-system$”                       | which namespaces will not have pods forwarded to YuniKorn for scheduling   							   |
+| `labelNamespaces`    | "" 								   | which namespaces will have pods labeled with an applicationId              							   |
+| `noLabelNamespaces`  | ""                                    | which namespaces will not have pods labeled with an applicationId          							   |
+| `generateUniqueAppId`| "false"                               | Whether to generate unique applicationId for all the apps that do not have an applicationId to start with |
+| `defaultQueue`       | "root.default"                        | Default queue name      																				   |
+If `processNamespaces` is setting is an empty string, pods created in all namespaces will be scheduled by YuniKorn, similarly, if `labelNamespaces` is setting is an empty string, all pods forwarded to YuniKorn will have an `applicationId` label applied.
+
+`bypassNamespaces` and `noLabelNamespaces` act as exception lists for processNamespaces and labelNamespaces respectively.
+
+If `generateUniqueAppId` is set to false, all the apps in a namespace should be bundled under a single applicationId, otherwise, the namespace and pod's uid will be used to generate a unique applicationId for it.
+
+Access control configuration, all entries start with the prefix `admissionController.accessControl.`.
+
+| Variable           | Default value                         | Description                                                                |
```

Comment:
These are already part of the Service Configuration. We should not have the same information in two locations as that becomes a sync issue. Better is to point back to the service config.